js/main.js

```diff
@@ -0,0 +1,19 @@
+const random = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+
+let str = prompt("Введите строоку " , "");
+let maxlegth = +prompt("Введите максимальную длинну " , "")
+if(str == undefined || str == ""){
+    alert("Строка не введена");
+}else{
+    truncate();
+}
+function truncate() {
```

Comment:
функция должна принимать строки и максимальную длину
```suggestion
function truncate(str, maxLength) {
```